crates/shared/src/http_solver.rs

```diff
@@ -117,7 +120,8 @@ impl HttpSolverApi for DefaultHttpSolverApi {
             .checked_sub(Duration::from_secs(1))
             .context("no time left to send request")?;
 
-        let mut url = self.base.join("solve").context("join base")?;
+        let path = self.custom_solve_path.as_deref().unwrap_or("solve");
```

Comment:
Maybe to be more explicit make `custom_solve_path` non optional and set it to "solve" everywhere?

Reply:
Good idea. This at least makes it more explicit and opens up the possibility of configuring it everywhere.

Reply:
Implemented this and made the `YEARN_SOLVER_PATH` configurable to facilitate the switch over when it happens.